### ISSUE TITLE
App works fine; just in one browser, but not in a different browser not in a different computer

### ISSUE THREAD

**Issue by blaimug** [0x5eba/Video-Meeting#20]
Dear @0x5eba, Thank you very much for the demo and the code, which has helped us all progress. The app, either in your demo, or our owns built on your own, it perfectly works in one browser, and the call is wonderful. However, on a different browser, and a different computer _it does not work, just a blank window._ I've tested different changes to no avail. Would it be possible to please assist? Is it an issue with STUN server, TURN server or certain resources or limitations? Thank you very much.

**Comment by 0x5eba** [comment]
It only works on chrome for now.
And for me it works using 3 different devices, so I don't know how to help you

**Comment by blaimug** [comment]
Dear @0x5eba. Thank you very much for your reply. Actually, I've retested both apps, your version on video.sebastienbiollo.com and mine hosted on heroku in Chrome, but it did not work either. However the chat feature works perfectly. Only the video call, the remote video call does not appear. I've tested both computers with Chrome version 85.0.4183, however it did not work. Is there something else, I would try, or a different environment?  Your assistance is greatly appreciated!
